.pre-commit-config.yaml

```diff
@@ -38,6 +38,38 @@ repos:
         args: ["--fix", "--show-fixes", "--exit-non-zero-on-fix"]
       - id: ruff-format
 
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: "v1.10.0"
+    hooks:
+      - id: mypy
+        additional_dependencies:
+          - types-aiofiles
+          - types-requests
+          - pandas-stubs
+          - types-pillow
+          - matplotlib
+        exclude: |
```

Comment:
I wonder if pre-commit could reuse the config and thus this exclude list from pyproject.toml?

Reply:
That's the part I could not get to work :/ I don't know why it doesn't!